### PR TITLE
🐛Source Asana: Add cash for project stream

### LINKED ISSUE #13676: Source Asana: Bad request (400) when syncing stories stream
## Environment
- **Airbyte version**:  0.35.47-alpha
- **OS Version / Instance**: Amazon Linux 5.10.102-99.473.amzn2.x86_64, AWS EC2
- **Deployment**: Docker
- **Source Connector and version**: Asana 0.1.3
- **Destination Connector and version**: BigQuery 1.1.6
- **Step where error happened**: Sync job

## Current Behavior
When syncing the stories stream, an error 400 is returned by Asana, causing the sync to fail.

## Expected Behavior
The sync should complete normally.

## Logs
[logs-3934.txt](https://github.com/airbytehq/airbyte/files/8878209/logs-3934.txt)

## Steps to Reproduce
1. Set up a connection between Asana and BigQuery (Asana validated with API token)
2. Select all streams for sync with `Full refresh | Overwrite`
3. Select `Normalized tabular data` for transformation settings

## Are you willing to submit a PR?
Not in the near future.

### PR DESCRIPTION
**Pull request by lazebnyi**
## What
#13676 - Source Asana: Bad request (400) when syncing stories stream
Offset token expired during pagination. 

## How
Add cash to project parent stream. 

## Pre-merge Checklist
<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- [ ] Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- [ ] Secrets in the connector's spec are annotated with `airbyte_secret`
- [ ] Unit & integration tests added and passing. Community members, please provide proof of success locally e.g: screenshot or copy-paste unit, integration, and acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
- [ ] Code reviews completed
- [ ] Documentation updated
    - [ ] Connector's `README.md`
    - [ ] Connector's `bootstrap.md`. See [description and examples](https://docs.google.com/document/d/1ypdgmwmEHWv-TrO4_YOQ7pAJGVrMp5BOkEVh831N260/edit?usp=sharing)
    - [ ] Changelog updated in `docs/integrations/<source or destination>/<name>.md` including changelog. See changelog [example](https://docs.airbyte.io/integrations/sources/stripe#changelog)
- [ ] PR name follows [PR naming conventions](https://docs.airbyte.io/contributing-to-airbyte/updating-documentation#issues-and-pull-requests)

</details>

### CODE REVIEW

**Comment by sherifnada** [comment]
@lazebnyi how did you conclude that the cursor expired?

**Comment by lazebnyi** [comment]
@sherifnada 
We get 400 error for project API during stories stream sync. So, the problem with project stream, after double check query params in url in error and confirming the correctness of the query params. Try to reproduce offset token expiration case, and get 400 error when offset token expired after less than 10-15 min. 

image

File: airbyte-integrations/connectors/source-asana/source_asana/streams.py [hunk 1]
```diff
@@ -141,6 +141,8 @@ def path(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> str:
 
 
 class Projects(WorkspaceRequestParamsRelatedStream):
+    use_cache = True
```
Comment (davydov-d) [inline airbyte-integrations/connectors/source-asana/source_asana/streams.py:144]: how does cache fix the issue? do we perform multiple identical requests within this stream?

Comment (lazebnyi) [reply airbyte-integrations/connectors/source-asana/source_asana/streams.py:144]: you can find more details in comment above

**Review by davydov-d — APPROVED** [review]

**Review by bazarnov — APPROVED** [review]

**Pull request merged**